11/11/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Data;
+using System.Data.Common;
+
+namespace Table
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите кол-во строк от 5 до 20");
+            int row = int.Parse(Console.ReadLine());
+            if (row < 5 || row > 20)
+                while (row < 5 || row > 20)
+                {
+                    Console.WriteLine("Ошибка! Введите значение от 5 до 20");
+                    row = int.Parse(Console.ReadLine());
+                }
+            Console.WriteLine("Введите кол-во столбцов от 5 до 20");
+            int column = int.Parse(Console.ReadLine());
+            if (column < 5 || column > 20)
+            {
+                while (column < 5 || column > 20)
+                {
+                    Console.WriteLine("Ошибка! Введите значение от 5 до 20");
+                    column = int.Parse(Console.ReadLine());
+                }
+            }
```

Comment:
Здесь можно опустить `{ }` в операторе `if`.
```suggestion
            if (column < 5 || column > 20)
                while (column < 5 || column > 20)
                {
                    Console.WriteLine("Ошибка! Введите значение от 5 до 20");
                    column = int.Parse(Console.ReadLine());
                }
```